doc/command_ref.rst

```diff
@@ -1599,36 +1596,53 @@ Upgrade-To Command
 Specifying Packages
 ===================
 
-Many commands take a ``<package-spec>`` parameter that selects a package for the
-operation. DNF looks for interpretations of the parameter from the most commonly
-used meanings to the least, that is it tries to see if the given spec fits one
-of the following patterns (in decreasing order of priority):
+Many commands take a ``<package-name-spec>`` parameter that selects a package
```

Comment:
Please do not describe `<package-name-spec>` on the first place but `<package-spec>`. `<package-spec>` is used by many commands.